util/parser.go

```diff
@@ -76,3 +76,11 @@ func FormatTimeToString(inputTime time.Time, format string) string {
 func DateTimeToRecordFormat(inputTime time.Time) *string {
 	return StringValueOrNil(FormatTimeToString(inputTime, RecordDateTimeFormat))
 }
+
+func DateTimeToRFC3339(inputTime time.Time) string {
+	if inputTime.IsZero() {
+		return ""
+	}
+
+	return inputTime.Format(time.RFC3339)
+}
```

Comment:
I would replace the body of this function to the following:

```go
return FormatTimeToString(inputTime, time.RFC3339)
```

Essentially it is duplicating what `FormatTimeToString` is doing, and since we already have that function, I would just use it!